pkg/apis/istio/v1alpha1/session_types.go

```diff
@@ -8,12 +8,27 @@ import (
 
 // SessionSpec defines the desired state of Session
 type SessionSpec struct {
-	Ref string `json:"ref,omitempty"`
+	Refs []string `json:"ref,omitempty"`
 }
 
 // SessionStatus defines the observed state of Session
 type SessionStatus struct {
-	State *string `json:"state,omitempty"`
+	State *string      `json:"state,omitempty"`
+	Refs  []*RefStatus `json:"refs,omitempty"`
+}
+
+// RefStatus defines the observed state of the individual Ref
+type RefStatus struct {
+	Name      string            `json:"name,omitempty"`
+	Params    map[string]string `json:"params,omitempty"`
```

Comment:
Would it make sense to elaborate on what `Params` are used for?

Reply:
https://github.com/aslakknutsen/istio-workspace/issues/60